collections/userEvents.coffee

```diff
@@ -61,3 +62,13 @@ Meteor.methods
         lastModifiedByUserId: user._id,
         lastModifiedByUserName: user.profile.name
       }})
+  updateUserEventArticleCount: (id, countModifier) ->
+    event = UserEvents.findOne(id)
```

Comment:
It seems that any user can add articles to an event right now, not just admins. Should I check for a user instead?